source/docs/03-metadata.xml

```diff
@@ -889,60 +889,32 @@
                   </specList>
                </p>
                <p>All the components of <gi scheme="MEI">work</gi> are optional, but they must occur in the following order:</p>
-               <list rend="numbered">
-                  <item>
-                     <gi scheme="MEI">identifier</gi>
-                  </item>
-                  <item>
-                     <gi scheme="MEI">title</gi>
-                  </item>
-                  <item>Responsibility-like elements including <gi scheme="MEI">composer</gi>, <gi scheme="MEI">lyricist</gi>, <gi scheme="MEI">contributor</gi> and others</item>
-                  <item>
-                     <gi scheme="MEI">incip</gi>
-                  </item>
-                  <item>
-                     <gi scheme="MEI">key</gi>
-                  </item>
-                  <item>
-                     <gi scheme="MEI">mensuration</gi>
-                  </item>
-                  <item>
-                     <gi scheme="MEI">meter</gi>
-                  </item>
-                  <item>
-                     <gi scheme="MEI">tempo</gi>
-                  </item>
-                  <item>
-                     <gi scheme="MEI">otherChar</gi>
-                  </item>
-                  <item>
-                     <gi scheme="MEI">history</gi>
-                  </item>
-                  <item>
-                     <gi scheme="MEI">langUsage</gi>
-                  </item>
-                  <item>
-                     <gi scheme="MEI">perfMedium</gi>
-                  </item>
-                  <item>
-                     <gi scheme="MEI">audience</gi>
-                  </item>
-                  <item>
-                     <gi scheme="MEI">contents</gi>
-                  </item>
-                  <item>
-                     <gi scheme="MEI">context</gi>
-                  </item>
-                  <item>
-                     <gi scheme="MEI">biblList</gi>
-                  </item>
-                  <item>
-                     <gi scheme="MEI">notesStmt</gi>
-                  </item>
-                  <item>
-                     <gi scheme="MEI">classification</gi>
-                  </item>
-               </list>
+               <p>
+                  <specList>
+                     <specDesc key="identifier"/>
+                     <specDesc key="title"/>
+                  </specList>
+               </p>
+               <p>Responsibility-like elements including <gi scheme="MEI">composer</gi>, <gi scheme="MEI">lyricist</gi>, <gi scheme="MEI">contributor</gi> and others</p>
```

Comment:
In the [current guidelines](https://music-encoding.org/guidelines/dev/content/metadata.html) this statement is part of the items list, but here it has been moved to its own paragraph. 

It also keeps the old `gi scheme` method of referring to elements, rather than moving to the `specDesc` method.

Perhaps this item needs to be re-phrased to fit into the new structure?

Reply:
I can't reconstruct why I've done this change. But I guess that I read this as an error or relict. I'm a bit irritated because my PR should not contain changes in section 3.6. If I'm wrong and this is not a bug, I would like to discard this change. It should not be necessary for restructuring section 3.7.
Thanks for this note :)